test/JDBC/expected/single_db/db_accessadmin-vu-verify.out

```diff
@@ -0,0 +1,337 @@
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+REVOKE CONNECT FROM babel_5136_u1_new_name
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+REVOKE CONNECT FROM dbo
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Grantor does not have GRANT permission.)~~
+
+
+-- NO OTHER DDLs SHOULD BE ALLOWED
+CREATE TABLE babel_5136_t1 (id INT)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for schema dbo)~~
+
+CREATE PROCEDURE babel_5136_p1 AS SELECT 1
+GO
+~~ERROR (Code: 2714)~~
+
+~~ERROR (Message: Function 'babel_5136_p1' already exists with the same name)~~
+
+CREATE FUNCTION f1() RETURNS INT AS BEGIN RETURN 1 END
+GO
+~~ERROR (Code: 2714)~~
+
+~~ERROR (Message: Function 'f1' already exists with the same name)~~
+
+CREATE DATABASE new_database
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Only one user database allowed under single-db mode. User database "babel_5136" already exists)~~
+
+
+-- In master database the current login should not have db_accessadmin privilege
+-- Since db_accessadmin is database level role
+USE master
+GO
+
+SELECT current_user
+GO
+~~START~~
+varchar
+babel_5136_user_master
+~~END~~
+
+
+-- should not be a member of db_accessadmin in master database
+SELECT IS_ROLEMEMBER('db_accessadmin')
+GO
+~~START~~
+int
+0
+~~END~~
+
+
+-- Should not be able to do any activity in master database
+CREATE USER babel_5136_u2 FOR LOGIN babel_5136_l2
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: User does not have permission to perform this action.)~~
+
+DROP USER babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the user 'babel_5136_u1', because it does not exist or you do not have permission.)~~
+
+DROP ROLE babel_5136_r1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot drop the role 'babel_5136_r1', because it does not exist or you do not have permission.)~~
+
+CREATE ROLE babel_5136_r2
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: User does not have permission to perform this action.)~~
+
+CREATE TABLE babel_5136_t1 (id INT)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for schema master_dbo)~~
+
+CREATE PROCEDURE babel_5136_p2 AS SELECT 1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for schema master_dbo)~~
+
+CREATE FUNCTION babel_5136_f2() RETURNS INT AS BEGIN RETURN 1 END
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for schema master_dbo)~~
+
+CREATE SCHEMA babel_5136_s1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for database jdbc_testdb)~~
+
+CREATE SCHEMA babel_5136_s1 AUTHORIZATION babel_5136_u1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: permission denied for database jdbc_testdb)~~
+
+CREATE DATABASE new_database
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Only one user database allowed under single-db mode. User database "babel_5136" already exists)~~
+
+
+
```

Comment:
Please make sure that
1. It is being tested in both the migration mode. 
2. we are tracking the testcase about the restrictions on the role that it should not be altered/dropped from either of endpoint as separate effort.
3. we test SSMS utility as well for visibility and usage of this new role via UI.
4. We are verifying that it should not be the owner of anything else apart from schema and schema-contained objects.